src/smc-webapp/editor_chat.cjsx

```diff
@@ -785,7 +790,9 @@ ChatRoom = (name) -> rclass
     on_scroll: (e) ->
```

Comment:
just as an idea: is `on_scroll` fired all the time? maybe we should wrap it in `throttle`.